xclim/indices/stats.py

```diff
@@ -729,10 +729,12 @@ def standardized_index_fit_params(
         "scipy_dist": dist,
         "method": method,
         "group": group,
-        "time_indexer": indexer,
         "units": "",
         "offset": offset,
     }
+    if indexer != {}:
+        params.attrs["time_indexer"] = str(indexer)
+
```

Comment:
I think having info on the indexer used could be useful, unless it's already part of some automatically generated history info, but I don't think so.  

For instance, if `indexer == {"date_bounds":("01-01","07-01")}`, then this would be saved as:
`"{'date_bounds': ('01-01', '07-01')}"`
It's a bit clumsy but at least it keeps information. Feedback on this?

Reply:
Ok, in short to recover indexer, we need: 
```python
        if indexer == "":
            indexer = {}
        else:
            indexer = {indexer[0]:indexer[1:]}
```
Is this really the first time this kind of operation in xclim is needed?